externs/olx.js

```diff
@@ -4975,14 +4974,6 @@ olx.source.ImageStaticOptions.prototype.imageExtent;
 
 
 /**
- * Size of the image in pixels.
- * @type {ol.Size|undefined}
- * @api stable
- */
-olx.source.ImageStaticOptions.prototype.imageSize;
```

Comment:
Not sure if anybody was using it this way, but if anybody is using an SVG as an image source, the size needs to be set explicitly in IE.  Edge case perhaps, but it would be good to at least document in the upgrade notes that the image size can no longer be set.